apollo-gradle-plugin-incubating/src/main/kotlin/com/apollographql/apollo/gradle/internal/ApolloGenerateSourcesTask.kt

```diff
@@ -18,6 +19,13 @@ abstract class ApolloGenerateSourcesTask : DefaultTask() {
   @get:Optional
   abstract val customTypeMapping: MapProperty<String, String>
 
+  @get:Internal
+  abstract val customIdGenerator: Property<CustomIdGenerator>
+
+  @Input
+  @Optional
+  fun customIdGeneratorVersion() = customIdGenerator.orNull?.version
```

Comment:
It looks like gradle requires Java Beans syntax:

```suggestion
  fun getCustomIdGeneratorVersion() = customIdGenerator.orNull?.version
```

Reply:
It's also what makes the gradle 6.0 tests fail:

```
Type 'ApolloGenerateSourcesTask': non-property method 'customIdGeneratorVersion()' should not be annotated with: @Input, @Optional. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
```